src/discriminant-calculator.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
+int discriminantCalculator(int a, int b, int c) {
+	int res = pow(b, 2) - 4 * a * c;
+	int x1 = (-b + sqrt(res)) / (2 * a);
+	int x2 = (-b - sqrt(res)) / (2 * a);
+	
+	printf("Discriminant = %d \n", res);
+	printf("x1 = %d\nx2 = %d \n", x1, x2);
+
+	if (res < 0) {
+		printf("Discriminant < 0, has 2 complex roots");
+	}
+
+	return 1;
```

Comment:
It is better to return the object with the status of the execution.
The object definitely should have the following fields:
- message
- result
- isSuccess

Reply:
done